docs/api/app.md

```diff
@@ -647,6 +647,27 @@ Sets the application's [dock menu][dock-menu].
 
 Sets the `image` associated with this dock icon.
 
+### `app.launcher.setBadgeCount(count)` _Linux_
+* `count` Integer
+
+Sets the number to be displayed next to the app icon in the unity launcher.
+Setting count to `0` will hide the badge.
+
+**Note:** This feature is currently only supported on Ubuntu Unity. Calling this function has no effect, when the application is running in a different environment.
+
+**Note:** you need to specify the .desktop file name to the desktopName field in package.json. By default, it will assume app.getName().desktop in packaged apps.
+
+### `app.launcher.getBadgeCount(count)` _Linux_
+
+Returns the current value displayed in the counter badge next to the launcher icon.
+
+**Note:** As `setBadgeCount` only supports Ubuntu Unity, the value will be 0 when the application is running in a different environment.
+
+### `app.launcher.isCounterBadgeAvailable()` _Linux_
```

Comment:
Since this returns the value of `unity::isRunning()` maybe this method should just be called `app.launcher.isUnityRunning()` so it becomes a general purpose check people can use to know when the launcher is available.